src/Classes/TreeTab.lua

```diff
@@ -91,6 +95,9 @@ local TreeTabClass = newClass("TreeTab", "ControlHost", function(self, build)
 		end
 	end)
 	self.controls.compareSelect.shown = false
+	self.controls.compareSelect.maxDroppedWidth = 1000
+	self.controls.compareSelect.enableDroppedWidth = true
+	self.controls.compareSelect.enableChangeBoxWidth = true
```

Comment:
This ends up cutting off the search box on smaller resolutions.  Since this only happens on the longest of tree names, and you can still use the search box by clicking on it (but you can't see what you type), I'll still allow it

Reply:
There are more drops down's than just ones i affected. We'll see how it goes.

pH